src/current/v24.3/ldap-authorization.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configure LDAP Authorization
+summary: Learn how to configure role-based access control (authorization) using LDAP with directory services like Active Directory.
+toc: true
+---
+
+LDAP authorization allows CockroachDB to use group memberships accessed using LDAP, for example, from an Active Directory server, as the source of truth for [role](https://www.cockroachlabs.com/docs/stable/security-reference/authorization#roles) assignments. When enabled:
+
+1. During login, CockroachDB retrieves the user's group memberships using LDAP.
+2. Groups are mapped to CockroachDB roles using the group's Common Name (CN).
+3. The user is automatically granted membership to corresponding roles.
+4. Any existing role memberships not matched to LDAP groups are revoked.
+
+## Prerequisites
+
+- Enable [LDAP Authentication]({% link v24.3/ldap-authentication.md %}).
```

Comment:
Is a configuration possible where the identities are not synced from LDAP (just normal CRDB users) but the roles are synced? This implies not.